src/Common/src/Interop/Unix/System.Net.Security.Native/Interop.NetSecurityNative.cs

```diff
@@ -0,0 +1,138 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static partial class NetSecurityNative
+    {
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_ReleaseBuffer")]
+        internal static extern Status ReleaseBuffer(
+            out Status minorStatus,
+            IntPtr bufferHandle);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_CopyBuffer")]
+        internal static extern Status CopyBuffer(
+            SafeGssBufferHandle handle,
+            byte[] buffer,
+            int capacity,
+            int offset);
+
+        [DllImport(Interop.Libraries.NetSecurityNative, EntryPoint="NetSecurityNative_DisplayStatus")]
+        internal static extern Status DisplayStatus(
+            out Status minorStatus,
+            Status statusValue,
```

Comment:
This value can be either a major status or a minor status. A single API for both major status and the minor status.